src/main/java/wolox/training/models/User.java

```diff
@@ -56,6 +56,9 @@
     public User() {
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.